ios/FluentUI/Core/Theme/Tokens/DynamicColor.swift

```diff
@@ -7,10 +7,7 @@ import SwiftUI
 
 /// A platform-agnostic representation of a 32-bit RGBA color value.
 public struct ColorValue {
-    var r: CGFloat { CGFloat((hexValue & 0xFF000000) >> 24) / 255.0 }
-    var g: CGFloat { CGFloat((hexValue & 0x00FF0000) >> 16) / 255.0 }
-    var b: CGFloat { CGFloat((hexValue & 0x0000FF00) >> 8) / 255.0 }
-    var a: CGFloat { CGFloat(hexValue & 0x000000FF) / 255.0 }
+    public static let clear: ColorValue = .init(r: 0.0, g: 0.0, b: 0.0, a: 0.0)
```

Comment:
Are these changes part of the Button changes? Seems like they belong in a separate PR?

Reply:
It's just code sanitization (moving `public` above `internal`), so I figured I could roll it into this PR. Would you prefer that I exclude this change and spin up a separate PR?